library/src/main/java/com/novoda/downloadmanager/MigrationExtractor.java

```diff
@@ -0,0 +1,63 @@
+package com.novoda.downloadmanager;
+
+import android.database.Cursor;
+
+import com.novoda.notils.logger.simple.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class MigrationExtractor {
+
+    private static final String BATCHES_QUERY = "SELECT batches._id, batches.batch_title FROM batches INNER JOIN DownloadsByBatch ON " +
+            "DownloadsByBatch.batch_id = batches._id WHERE DownloadsByBatch.batch_total_bytes = DownloadsByBatch.batch_current_bytes";
+    private static final String URIS_QUERY = "SELECT uri, _data, total_bytes FROM Downloads WHERE batch_id = ?";
+    private static final int FIRST_COLUMN = 0;
+    private static final int SECOND_COLUMN = 1;
+    private static final int THIRD_COLUMN = 2;
```

Comment:
FIRST_COLUMN isn't really any more meaningful to me than 0. Maybe like COLUMN_INDEX_URI, or URI_COLUMN, or something (depending on your pref for grouping hierarchical names)? i.e. mention column name